modules/installation-azure-marketplace-subsribe.adoc

```diff
@@ -15,34 +15,51 @@ endif::[]
 :_content-type: PROCEDURE
 [id="installation-azure-marketplace-subscribe_{context}"]
 = Selecting an Azure Marketplace image
-If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes.
+If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes. When obtaining your  image, consider the following:
+
```

Comment:
`your  image` has more white space ?

Reply:
Thank you. I removed the extra space. Not sure why this comment does not show the update, but the space has been removed and the published output looks good. [1]

Appreciate your feedback. Is there anything else or is the doc good to go?

[1] http://file.rdu.redhat.com/mpytlak/osdocs-3029/installing/installing_azure/installing-azure-customizations.html#installation-azure-marketplace-subscribe_installing-azure-customizations